api-flask/app/generate_jwt.py

```diff
@@ -0,0 +1,55 @@
+import json
```

Comment:
@JorgeMartinezG we could repurpose that to generate a list of users with passwords and add them to the DB

Reply:
The bbox needs to be entered into the "acccess" json:

we then get is as `user_info["access"]["bbox"]`